prepare-image.sh

```diff
@@ -8,3 +8,7 @@ dnf upgrade -y
 dnf --setopt=install_weak_deps=False install -y $(cat /tmp/main-packages-list.txt)
 dnf clean all
 rm -rf /var/cache/{yum,dnf}/*
+if [ -s "/tmp/${PATCH_LIST}" ]; then
```

Comment:
This test returns true for directories
```
+ '[' -s /tmp/ ']'                                              
+ /bin/patch-image.sh     
+ patch_file=/tmp/
+ IFS=
+ read -r line
/bin/patch-image.sh: line 6: read: read error: 0: Is a directory

```

Reply:
yes, I forgot to correct this one, thanks!